Update snowflake-qs.md (#6763)
dbt-labs/docs.getdbt.com: website/docs/guides/snowflake-qs.md

## Create a new Snowflake worksheet 
1. Log in to your trial Snowflake account. 

> 2. In the Snowflake UI, click **+ Create** in the left-hand corner, underneath the Snowflake logo to create a new worksheet. This will open a dropdown. Select the first option, “SQL Worksheet”. 

Reviewer: simplifying a tiny bit and using bold:
```suggestion
2. In the Snowflake UI, click **+ Create** in the left-hand corner, underneath the Snowflake logo, which opens a dropdown. Select the first option, **SQL Worksheet**. 
```
